### PR TITLE
Use JDK11 on travis

### PR DESCRIPTION
**Pull request by jdneo**
Signed-off-by: Sheng Chen <sheche@microsoft.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ matrix:
   include:
     - os: linux
       dist: trusty
+      jdk: openjdk11
     - os: osx
```
Comment (fbricon) [inline .travis.yml:12]: why not setting the jdk for mac too?

**Comment by fbricon** [comment]
> The command "$TRAVIS_BUILD_DIR/install-jdk.sh --install openjdk11 --target $JAVA_HOME" exited with 127.

**Comment by jdneo** [comment]
Finally....... 🤣🤣🤣

**Comment by fbricon** [comment]
Thanks @jdneo !

**Pull request merged**